web/app/js/components/TopEventTable.jsx

```diff
@@ -21,6 +21,11 @@ const topColumns = (resourceType, ResourceLink, PrefixedLink) => [
     width: "180px",
     render: d => srcDstColumn(d, resourceType, ResourceLink)
   },
+  {
+    title: "Method",
+    dataIndex: "httpMethod",
+    sorter: (a, b) => a.httpMethod.localeCompare(b.httpMethod),
+  },
```

Comment:
I'm inclined to put a width of 98px to this column, to bring it closer to the path column, since they go hand-in-hand.

Reply:
good point!